OpenOversight/app/models.py

```diff
@@ -632,6 +632,11 @@ def change_email(self, token):
         db.session.add(self)
         return True
 
+    @property
+    def is_active(self):
+        """Override UserMixin.is_active to prevent disabled users from logging in."""
+        return not (self.is_disabled)
```

Comment:
Are the parens necessary here? They're a little surprising (I didn't expect them at least) and made me wonder if there was a missing and/or that was intended.

```suggestion
        return not self.is_disabled
```